code/_onclick/click.dm

```diff
@@ -230,7 +230,7 @@
 			if (!target.loc)
 				continue
 
-			if(target.loc.atom_storage)
+			if(target.loc?.atom_storage)
```

Comment:
You don't need this one here, loc is already known to not be null due to the if above